### PR TITLE
[ICDS Dashboard]: Fixed AWC Activity report

### PR DESCRIPTION
**Pull request by ShubhamBansal1997**
##### SUMMARY
https://dimagi-dev.atlassian.net/browse/QA-1412
https://dimagi-dev.atlassian.net/browse/QA-1427

### CODE REVIEW

File: custom/icds_reports/sqldata/exports/aww_activity_report.py [hunk 1]
```diff
@@ -54,12 +54,18 @@ def _format_date(data):
...
                 row['supervisor_name'],
                 row['awc_name'],
-                row['awc_site_code'],
-                _format_date(_format_infrastructure_data(row['first_submission'])),
-                _format_date(_format_infrastructure_data(row['last_submission'])),
-                _format_infrastructure_data(row['no_of_days_since_start']),
-                _format_infrastructure_data(row['no_of_days_inactive'])
+                row['awc_site_code']
             ]
+            if row['first_submission'] is None or row['first_submission'] == '':
```
Comment (czue) [inline custom/icds_reports/sqldata/exports/aww_activity_report.py:59]: nit: could be replaced with `if not row['first_submission']`

**Review by czue — APPROVED** [review]

File: custom/icds_reports/sqldata/exports/aww_activity_report.py (hunk 1 above)
Comment (ShubhamBansal1997) [reply custom/icds_reports/sqldata/exports/aww_activity_report.py]: updated!

**Review by czue — APPROVED** [review]

**Review by czue — APPROVED** [review]
only reviewed https://github.com/dimagi/commcare-hq/pull/27836/commits/2c82fc8969345271a0c8a8d531af3501e07e6735 since the last review

**Comment by ShubhamBansal1997** [comment]
Thanks @czue 

Build Passed https://travis-ci.org/github/dimagi/commcare-hq/builds/700765422?utm_source=github_status&utm_medium=notification
Moving PR in

**Pull request merged**